src/docs/asciidoc/getting_started.adoc

```diff
@@ -16,14 +16,33 @@ You can use the Hazelcast
 link:https://github.com/hazelcast/hazelcast-command-line[Command Line Interface (CLI)^]
 to install and start Hazelcast IMDG.
 
-Assuming you have the link:https://brew.sh/[Homebrew^] package manager, run the following commands:
+Run the following commands to install Hazelcast CLI:
 
-[source,bash,subs="attributes+"]
+[source,bash,indent=0,subs="verbatim,attributes",role="secondary"]
+.Homebrew
 ----
 $ brew tap hazelcast/hz
```

Comment:
Before the actual brew, Debian and RPM commands, don't we need to mention that the user already needs to have the related package manager installations? (maybe giving links to the respective manager, in each code tab, before the actual commands)

Reply:
This package managers are essential parts of the distributions. I think it's completely safe to not provide the installation instruction links. I have never seen this in any other manual :)

Reply:
My concern was lack of requirements for the OS. What we had by the time I was writing this is instruction for homebrew, which is only applicable to MacOS, while we have Windows and Linux users.